src/main/java/com/exadel/discount/service/DiscountServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.exadel.discount.service;
+
+import com.exadel.discount.entity.Discount;
+import com.exadel.discount.repository.DiscountRepository;
+import com.exadel.discount.service.interfaces.DiscountService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@Service
+public class DiscountServiceImpl implements DiscountService {
+
+    private final DiscountRepository discountRepository;
+
+    @Override
+    public Discount create(Discount discount) {
+        return discountRepository.save(discount);
+    }
+
+    @Override
+    public Discount get(UUID id) {
+        return discountRepository.getOne(id);
+    }
```

Comment:
Would U change getOne(id) to FindById(id)?

Reply:
+